src/navigations/root-stack.js

```diff
@@ -348,6 +349,11 @@ const AuthenticatedNavigator = () => {
           component={NewsScreen}
           options={{headerShown: false}}
         />
+        <AuthenticatedStack.Screen
+          name="CreateCommunity"
+          component={withSafeAreaView(withKeyboardWrapper(CreateCommunity))}
+          options={{headerShown: false}}
+        />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `CreateCommunity` screen to the navigation stack looks good. However, I would recommend adding some comments here to explain what `withSafeAreaView` and `withKeyboardWrapper` are doing for future maintainability. It's not immediately clear from the function names alone.
```diff
+         // Wrap CreateCommunity with safe area view and keyboard wrapper for better UX
          <AuthenticatedStack.Screen
            name="CreateCommunity"
            component={withSafeAreaView(withKeyboardWrapper(CreateCommunity))}
            options={{headerShown: false}}
          />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->